### PR TITLE
add a test for the transaction saga

### PR DESCRIPTION
**Pull request by gl2748**
This is fallout from the work on PR #1771.
A test for the transaction saga.

### CODE REVIEW

File: src/app/redux/TransactionSaga.test.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+import {
+    preBroadcast_comment,
+    createPermlink,
+    createPatch
+} from './TransactionSaga';
+import {DEBT_TICKER} from 'app/client_config';
+
+const operation = ({
+    type: 'comment', 
+    author : "Alice",
```
Comment (bnchdrff) [inline src/app/redux/TransactionSaga.test.js]: did prettier run on this file?

Comment (gl2748) [reply src/app/redux/TransactionSaga.test.js]: no, my b.

File: src/app/redux/TransactionSaga.test.js [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    preBroadcast_comment,
+    createPermlink,
+    createPatch,
+    watchForBroadcast,
+    watchForUpdateAuthorities,
+    watchForUpdateMeta,
+    watchForRecoverAccount,
+    preBroadcast_transfer,
+} from './TransactionSaga';
+import * as transactionActions from 'app/redux/TransactionReducer';
```
Comment (bnchdrff) [inline src/app/redux/TransactionSaga.test.js]: doesn't look like this is being used

File: src/app/redux/TransactionSaga.test.js [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+    successCallback,
+    parent_author,
+    parent_permlink,
+    type,
+    __config,
+    json_metadata,
+    title,
+    body,
+    memo,
+} = operation;
```
Comment (bnchdrff) [inline src/app/redux/TransactionSaga.test.js]: for clarity i think it'd be better to refer back to `operation` for these items in the tests below

File: src/app/redux/TransactionSaga.test.js [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+    watchForBroadcast,
+    watchForUpdateAuthorities,
+    watchForUpdateMeta,
+    watchForRecoverAccount,
+    preBroadcast_transfer,
+} from './TransactionSaga';
+import * as transactionActions from 'app/redux/TransactionReducer';
+import { DEBT_TICKER } from 'app/client_config';
+
+let operation = {
```
Comment (bnchdrff) [inline src/app/redux/TransactionSaga.test.js]: this can be a `const`

**Pull request merged**